src/github/interface.ts

```diff
@@ -1,14 +1,8 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/

```

Comment:
For
> Close button is remain after merge

I think the problem is that the return of `mergePullRequest` is different than `closePullRequest`. The `Promise<any>` below should be `Promise<IPullRequest>`. The return of merge seems to look like

```ts
data: {
	merged: boolean;
	message: string;
	sha: string;
}
```

I think we should create an interface for this data, and return result.data within the merge method.